### ISSUE TITLE
fix bugs with classification

### ISSUE THREAD

**Issue by verenich** [Zukkari/nirdizati-training-ui#103]
In multi-bucketing settings (bucketing method != `zero`), rare labels may be missing in some training buckets, therefore they (their probabilities) are never predicted in test set. Thus, prediction results from different buckets will have a different number of columns, and it will be problematic to concatenate them.

Solution: for each bucket, compare the set of predicted labels with the set of labels in the training set ("universe of all possible labels"). If the former has smaller size, complement it to the size of the latter. Complemented labels will have zero probabilities.
